src/Tribe/Views/V2/Views/Day_View.php

```diff
@@ -22,6 +22,14 @@ class Day_View extends View {
 	 */
 	protected $slug = 'day';
 
+	/**
+	 * Visibility for this view.
+	 *
+	 * @since TBD
+	 *
+	 * @var bool
+	 */
+	protected $publicly_visible = true;
```

Comment:
Should this variable be moved to the parent `View` class with a default of being set to `true`? Then we don't need to declare it in all of the child classes. It feels like a default of `true` is better than a default of `false` for folks that are trying to create views.

Perhaps your intent was to have it in all of the child classes so that folks who copy these View classes know that is a setting. Either way, we should probably still declare it in the parent so that `View::is_publicly_visible()` doesn't throw a warning if someone extends and doesn't declare this variable.

Reply:
Resolved.